streamlit_app.py

```diff
@@ -0,0 +1,608 @@
+
+Aside from the crucial role energy storage plays in the energy transition, energy storage can utilize the energy
+ markets to generate revenue.
+By utilizing powerful algorithms and strategies, energy storage can make trades and sell their capacity to the
+ highest bidder.
+They can earn revenue through price arbitrage for example.
+Utilizing wholesale markets to buy energy when the price is low, and sell it at a later moment when prices spike.
+Or energy storage can sell their flexible capabilities directly to system operators in so-called capacity markets.
+
+[GIGA Storage]({GIGA_HOME_PAGE}) develops, realizes and operates large-scale (Battery) Energy Storage Systems ([B]ESS).
+In the competitive and dynamic environment which the energy market is, it is essential for market participants
+ to benchmark and optimize their performance.
+A lot of optimisation comes from choosing the right energy market at the right time.
+Making use of available assets and market fluctuations.
+To challenge the energy community, GIGA Storage strives to develop the Battery Trading Benchmark.
+This will be an advanced optimisation model to determine the mathematical optimum of how a BESS could have been deployed
+ in the energy markets.
+This supports market participants to maximize their capture rate in the energy market and realize more BESS projects.
+
+This blogpost is written for anybody interested in (Battery) Energy Storage Systems and how they earn revenue on the
+ energy market.
+It is part of the open-source repository
+ [GigaStorage/battery-trading-benchmark]({REPOSITORY_HOME_PAGE})
+ that strives to be the hub for benchmarking how a BESS performs in the current state of the energy market.
+In this blogpost, we will deep dive into the first version of this benchmarking tool, and the iterative process taken
+ to develop it.
+By offering insight into the development of this benchmark, GIGA Storage hopes to contribute to a better understanding
+ of how BESS can help the energy transition and what this means for market participants.
+As such, this blog post will not deep-dive into the technical details of the developed model.
+For follow-up information regarding technical details,
+ [feel free to join our open-source community]({REPOSITORY_HOME_PAGE}).
+""")
+
+st.write("""
+### Define your own BESS
+This blogpost is written using [Streamlit](https://streamlit.io/).
+This means there is Python code running and generating these visuals right now.
+As such, you are free to define the limits of your own BESS.
+By default, a 2 hour system with 1 MW and 87% Round-Trip Efficiency is configured.
+If you are unsure what to configure for the other parameters, feel free to skip this part, and read on.
+""")
+
+# ---------- USER INPUT ----------
+# 1. Define the limits of your BESS
+col1, col2, col3, col4, col5, col6, col7 = st.columns(7)
+with col1:
+    max_power_kw = st.number_input("BESS Power (kW)", 100, 10000, value=1000, step=100)
```

Comment:
Ik zou graag ieder formaat BESS willen kunnen toevoegen, is het mogelijk dat er geen limieten worden gesteld op vermogen en capaciteit?